docs/source/using/troubleshooting.adoc

```diff
@@ -11,6 +11,28 @@ toc::[]
 
 This section contains solutions to common problems that you might encounter while using Minishift.
 
+[[minshift-update-failed-due-to-permission-denied]]
+== Permission denied error when updating Minishift
+
+When updating Minishift using `minishift update`, the update process needs write permissions for the Minishift binary as well as the directory in which it is located.
+Without these permisions `minishift update` will fail.
+This is for example an issue when installing Minishift as root.
```

Comment:
"For example, this issue might occur when installing Minishift as root." (grammar/syntax fix)